cli/flox-rust-sdk/src/models/environment/path_environment.rs

```diff
@@ -85,7 +85,7 @@ pub struct InitCustomization {
     pub profile_fish: Option<String>,
     pub profile_tcsh: Option<String>,
     pub profile_zsh: Option<String>,
-    pub packages: Option<Vec<PackageToInstall>>,
+    pub packages: Option<Vec<CatalogPackage>>,
```

Comment:
nit: i think this should stay `PackageToInstall` but i understand we'd practically only use this for `CatalogPackages`, feel free to ignore.